### PR TITLE
chore: upgrade wasmbuild to 0.6

### PR DESCRIPTION
**Pull request by dsherret**
- Uses new `--js-ext mjs` flag to output an `mjs` extension
- Uses new `--check` flag for verifying the output is up to date (works across operating systems)

### CODE REVIEW

**Review by kt3k — APPROVED** [review]
Oh, this is hugely simplified. Nice!

LGTM

File: _wasm_crypto/README.md [hunk 1]
```diff
@@ -11,10 +11,3 @@ deno task build:crypto
 ```
 
 This will regenerate the files in the `./lib/` folder from the Rust source.
-
-### CI
-
-If CI fails in `Verify Wasm hasn't changed` step, you need to download built
```
Comment (kt3k) [inline _wasm_crypto/README.md:17]: Does this mean there is no diffs anymore in the built wasm based on the platforms?

Comment (dsherret) [reply _wasm_crypto/README.md:17]: Yeah there’s no diffs. If the inputs are the same then the CI will pass. If someone builds on Windows then the output will be slightly different than if someone build on Linux, but the output should be functionally the same. So if the CI fails they should only have to run the deno task regardless of their OS, then recommit and it should pass the CI.

Comment (kt3k) [reply _wasm_crypto/README.md:17]: Thanks for the clarification! Makes sense

**Pull request merged**